File: python/svix/models/sns_config.py
# this file is @generated
import typing as t

from .common import BaseModel


class SnsConfig(BaseModel):
    """Configuration for a SNS sink."""

    topic_arn: str

    region: str

    access_key_id: str

    secret_access_key: str

    endpoint_url: t.Optional[str] = None

File: kotlin/lib/src/main/kotlin/models/SnsConfig.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class SnsConfig(
    val topicArn: String,
    val region: String,
    val accessKeyId: String,
    val secretAccessKey: String,
    val endpointUrl: String? = null,
)
